subcommand/acl-controller/command_test.go

```diff
@@ -223,17 +223,24 @@ func checkConsulResources(t *testing.T, consulClient *api.Client, expPolicyRules
 	// Check the client policy is created.
 	policies, _, err := consulClient.ACL().PolicyList(nil)
 	require.NoError(t, err)
-	sort.Slice(policies, func(i, j int) bool {
-		return policies[i].Name < policies[j].Name
-	})
 
-	require.Equal(t, policies[0].Name, "consul-ecs-client-policy")
+	policyNames := []string{}
+	for _, policy := range policies {
+		policyNames = append(policyNames, policy.Name)
+	}
```

Comment:
I changed this bit to collect the policy names. Then we do a contains check and the order of policies doesn't matter.